src/Components/HandPicked/HandPicked.js

```diff
@@ -1,14 +1,28 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
 import PropTypes from 'prop-types';
 import SingleTextCard from '../SingleTextCard/SingleTextCard';
 import { Grid, Box, Typography } from '@mui/material';
+import { fetchData } from '../../Services/network';
 
-HandPicked.propTypes = {
-    headLine: PropTypes.string.isRequired,
-    collections: PropTypes.array.isRequired
-};
+function HandPicked({ headLine }) {
+    const [collections, setCollections] = useState([]);
+    const [loading, setLoading] = useState(true);
+    const [error, setError] = useState('');
+
+    useEffect(() => {
+        const fetchDataAsync = async () => {
+            setLoading(true);
+            try {
+                const result = await fetchData('https://store-osn9.onrender.com/products/handpicked');
```

Comment:
unnecessary base url in the request URL,
you do it like this:
fetchData('/products/handpicked');